setup.py

```diff
@@ -1,11 +1,12 @@
 from setuptools import setup
-from eq_translations import __version__
+
+__version__ = "4.6.1"
```

Comment:
I think this should be available via `__init__`, as it common for packages to expose `__version__`.

Reply:
Don't think it is required, tests seem to fail now.

Reply:
I'm going to revert it back , I haven't seen any rule to expose `__versions__` it as `__all__`  for packages. It is not being used anywhere within package also. I'd suggest to remove it as we are going to move to poetry anyways which will get rid of `setup.py`. It also appears to be circular import if we include versions in `eq-translations/__init__ directory`. Let me know your thoughts

Reply:
https://www.python.org/dev/peps/pep-0396/. But ya okay, if it causing an issue right now, just do what needs to be done and we'll revist once we migrate to poetry. We may not even make use of this repo for a while.